sample-app/src/components/SearchBar.tsx

```diff
@@ -19,17 +20,20 @@ interface Props {
  */
 export default function SearchBar({ placeholder, isVertical, screenReaderInstructionsId }: Props) {
   const answersActions = useAnswersActions();
-  const query = useAnswersState(state => state.query.query);
-  const mapStateToAutocompleteResults: StateSelector<AutocompleteResult[] | undefined> = isVertical
-    ? state => state.vertical.autoComplete?.results
-    : state => state.universal.autoComplete?.results;
-  const autocompleteResults = useAnswersState(mapStateToAutocompleteResults) || [];
-  const isLoading = useAnswersState(state => state.vertical.searchLoading || state.universal.searchLoading);
-
-  function executeAutocomplete () {
+  const query = useAnswersState(state => state.query.input);
+  const [ autocompleteResults, setAutoCompleteResults ] = useState<AutocompleteResult[]>([]);
+  const isLoading = useAnswersState(state => state.searchStatus.isLoading);
+  const networkIds = useRef({ request: 0, response: 0 });
```

Comment:
what do you think about `autocompleteNetworkIds`? We could also update `request` to `latestRequest`

Reply:
I'm also wondering if we should update `response` to something like `responseInState` or `responseInUse`

Reply:
sounds good